### DISCUSSION TITLE
Set & use a variable as a connection name

### DISCUSSION THREAD

**Discussion by misi01** [dbeaver/dbeaver#17372] [General]
At our installation, for the sake of argument, we have 2 connections called AFRED and BFRED.
When running DBEAVER, our selections are ALWAYS something like: -

select * from xxxx.TABLE1

where xxxx is always AFRED or BFRED.

Is there some sort of variable we can use whereby the variable is changed automatically to the current connection. For example

select * from &connection.TABLE1

(on the assumption that &connection is a DBEAVER variable that is always set to the current connection name)

Note that I'm NOT looking for an example where I have to set a variable first, since it's easier to simply change the first character in the connection name myself.
Any suggestions?

**Comment by E1izabeth** [comment]
There is no such opportunity.

**Reply by misi01** [reply]
Thank you. It was worth asking  :-)
